src/PaymentGateways/PayPalCommerce/ScriptLoader.php

```diff
@@ -76,18 +78,34 @@ function givePayPalOnBoardedCallback(authCode, sharedId) {
 	 * @since 2.8.0
 	 */
 	public function loadPublicAssets() {
+		if ( ! Utils::isConnected() ) {
+			return;
+		}
```

Comment:
This helper function doesn't exist anymore. You'll need to use the `MerchantDetails` repository.

Reply:
fixed: https://github.com/impress-org/givewp/pull/5021/commits/dd79ab0fbe1020fe27e0df2f6dbc2c1a622307b7